component/credentialstatus/credentialstatus_service.go

```diff
@@ -421,7 +423,9 @@ func (s *Service) createVC(vcID string,
 
 // prepareSigningOpts prepares signing opts from recently issued proof of given credential.
 func prepareSigningOpts(profile *vc.Signer, proofs []verifiable.Proof) ([]vccrypto.SigningOpts, error) {
-	var signingOpts []vccrypto.SigningOpts
+	signingOpts := []vccrypto.SigningOpts{
```

Comment:
Signing of JWT VC doesn't care if it is SD-JWT or regular one. It is just signing a payload, let me know if I am missing anything here

Reply:
Agree, good point. 
I added option `WithEnabledSDJWT` to be able explicitly disable SD-JWT. It might be used in credential status service while creating CSL.
Do you think it's not necessary ?

Reply:
CSL has its own credential type which will never be SD-JWT, not sure if you need this flag.

Reply:
Each profile has it's [VC format](https://github.com/trustbloc/vcs/blob/main/test/bdd/fixtures/profile/profiles.json#L36). As we aligned, I will add a new object in profile struct that holds SD-JWT configuration.
So it's possible scenario that issuer profile has JWT VC format and SD-JWT enabled.

When such issuer profile will call Crypto package to sign credential, Crypto will return signed credential in SD-JWT format with disclosed subject claims. And it can happen in credential status service while creating CSL. TO cover such cases I added `WithEnabledSDJWT` option to explicitly turn it on/off. WDYT?

Reply:
Signer should never add disclosures - it just cares about adding signature.